integration_tests/class_01.py

```diff
@@ -0,0 +1,28 @@
+from lpython import i32,i64
+
+def fn_2():
+    print("Inside fn_2")
+    return
+ 
+class Test:
+    mem : i64 = i64(5)
+    s   : str = "abc"
+    def fn_1():
```

Comment:
Print are not being test, so I suggest to pass some arguments to the functions and check the computed return values.